easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -14,15 +14,26 @@ toolchainopts = {'usempi': True}
 # Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
 sources = [SOURCELOWER_TAR_GZ]
 
+patches = ['patch.5.4.1.08072015', 'patch.5.4.1.27082015', 'patch.5.4.1.06112015']
```

Comment:
@migueldiascosta please reformat to have one per line, and add a comment above to explain where these patches can be obtained from

```python
patches = [
    'patch.5.4.1.08072015',
    ...
]
```
Did you test this easyconfig btw, because I think EasyBuild requires that patch files are named `*.patch` (which is also the reason why the tests don't complain about not having the patches available next to the easyconfig file).

I'm not sure if we should start updating existing VASP easyconfigs with official patches, I'd rather just add a new easyconfig when a new VASP version is defined, and keep the builds as close to the official version as possible...